config/config_test.go

```diff
@@ -62,3 +64,61 @@ func TestResolveAddr(t *testing.T) {
 		}
 	}
 }
+
+func TestMakeOnces(t *testing.T) {
+	directives := []directive{
+		{"dummy", nil},
+		{"dummy2", nil},
+	}
+	directiveOrder = directives
```

Comment:
You could just assign the variable directly, I think (same with below). But I'll still merge this!